books/models.py

```diff
@@ -994,7 +997,7 @@ class BookIndex(Page):
 
     @property
     def books(self):
-        books = Book.objects.live().filter(locale=self.locale).order_by('title')
+        books = Book.objects.live().filter(locale=self.locale).filter(self.book_state is not 'unlisted').order_by('title')
```

Comment:
The more "Django" way to do this query would be 
`books = Book.objects.live().filter(locale=self.locale).exclude(self.book_state='unlisted').order_by('title')`

Reply:
PyCharm did not like this. Displayed an error: "Cannot assign to function call". I'm guessing it is related to calling live() before the filters